crates/store/re_mcap/src/parsers/ros2msg/util.rs

```diff
@@ -0,0 +1,10 @@
+/// Helper for suffixing image frame IDs with "_plane".
+///
+/// This is required to match the Rerun model for named pinhole frames, where the image plane has its own frame ID
+/// different from the pinhole frame. In ROS, both image and camera info share the same frame ID.
+pub fn suffix_image_plane_frame_ids(frame_ids: impl IntoIterator<Item = String>) -> Vec<String> {
+    frame_ids
+        .into_iter()
+        .map(|id| format!("{id}_plane"))
```

Comment:
"_plane" is a bit nicer than "_image" IMO, because most frame IDs usually end with "_image" already. Also if we add support for video etc it would be better.

Reply:
Actually, I like "image_plane" more 😆 ...